src/views/ajax.js

```diff
@@ -0,0 +1,112 @@
+const util = require('../lib/util');
+const dom = require('../lib/dom');
+
+let count = 0;
+let ready = false;
+let preRenderCache = [];
+let currentTimout = undefined;
+const summaryStack = [];
+
+function rowTemplate(method, uri, duration) {
```

Comment:
Yep... looking at making that change you suggested earlier re object and passing through.